internal/pkg/jwt/jwt.go

```diff
@@ -12,17 +12,17 @@ import (
 type (
 	Customeclaims struct {
 		jwt.StandardClaims
-		Useid       string
-		Usefullname string
-		Userole     string
+		Useid          string
```

Comment:
Please use camel case for naming. I have asked for this from last pull request but not sure why it's not fixed yet.